website/docs/secrets/manage-secrets-ui.mdx

```diff
@@ -61,9 +95,18 @@ The details page displays the details of a specific external secret, including i
 - **Secret Store:** This shows the name of the secret store provider that is being used to store the external secret.
 - **Secret Store Type:** This indicates the type of secret store that is being used to store the external secret. In this case, the type is "AWS Secrets Manager".
 - **Secret path:** This is the path to the external secret within the secret store.
-- **Property:** This is the property or key that is associated with the external secret.
 - **Version:** This shows the version of the external secret, which may be blank if no version has been specified.
 
+- #### Properties
+
+  Based on the configuration of the external secret, this section will vary:
+
+- If the "Include all properties" option was selected during the creation of the external secret, this section will display the text "All properties are included".
```

Comment:
Please take a look at the screenshoot: it seems to be test data for development of the feature. Given that this is end-user documentation, it might be better to have pictures that would reference some realistic scenario to help improve user understanding.